.github/workflows/build_and_test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
         # selected operation systems to run CI
         os: [ubuntu-latest] #, windows-latest, macos-latest]
         # selected node version to run CI
-        node-version: [20.10.x]
+        node-version: [20.11.1]
```

Comment:
I decided to revert to LTS version. One of the issues with that version was that it didn't work with globstar patterns (see https://github.com/nodejs/node/issues/44023). However, I found a solution by using the [node-glob](https://github.com/isaacs/node-glob) package, which is lightweight and constantly updated.

Reply:
Nice finding, thank you for doing the investigation 🙂